services/tasks/service.go

```diff
@@ -332,16 +332,30 @@ func (s *Service) Kill(ctx context.Context, r *api.KillRequest) (*google_protobu
 }
 
 func (s *Service) ListPids(ctx context.Context, r *api.ListPidsRequest) (*api.ListPidsResponse, error) {
+
 	t, err := s.getTask(ctx, r.ContainerID)
 	if err != nil {
 		return nil, err
 	}
-	pids, err := t.Pids(ctx)
+	processList, err := t.Pids(ctx)
 	if err != nil {
 		return nil, errdefs.ToGRPC(err)
 	}
+	var processes []*task.ProcessInfo
+	for _, p := range processList {
+		var processInfo task.ProcessInfo
+		processInfo.Pid = p.Pid
+		if p.Info != nil {
+			a, err := typeurl.MarshalAny(p.Info)
+			if err != nil {
+				log.G(ctx).Error(err)
```

Comment:
`log.G(ctx).WithError(err).Errorf("...something about  marshaling info for pid")`

Is `a` always nil in this case?

Reply:
yes `a` is nil